crates/re_renderer/src/renderer/depth_cloud.rs

```diff
@@ -242,6 +251,12 @@ impl DepthCloudDrawData {
             depth_cloud_ubo_binding_outlines,
             depth_cloud_ubo_binding_opaque
         ) {
+            if depth_cloud.depth_texture.format() != wgpu::TextureFormat::R32Float {
```

Comment:
R16Float should also be fine, right?

Reply:
true! Any format that samples as float arguably, even multichannel 🤔 (will just ignore everything but one channel!)
Fixing that